packages/api/src/lib/sendMail.ts

```diff
@@ -0,0 +1,37 @@
+import { Resend } from "resend";
+
+const resend = new Resend(process.env.RESEND_API_KEY);
+
+interface EmailDetails {
+  to: string;
+  name: string;
+}
+
+// Function to check if the mail service is configured correctly
+const canSendMail = (): boolean => {
+  return Boolean(process.env.RESEND_API_KEY);
+};
+
+export const sendMail = async ({ to, name }: EmailDetails) => {
```

Comment:
I want a generic function that will take inputs below:

- to (mail ID)
- subject
- body